concreate/descriptor.py

```diff
@@ -195,13 +195,60 @@ def __init__(self, descriptor):
         map:
           repositories:
             seq:
-              - {type: str}
+              - {type: any}
           install:
             seq:
-            - {type: str}""")]
+              - {type: str}"""),
+                        yaml.safe_load("""
+        map:
+          repositories:
+            seq:
+              - {type: any}
+          packages:
+            seq:
+              - {type: any}""")]
 
         super(Packages, self).__init__(descriptor)
 
+        if 'install' in self.descriptor:
+            self.descriptor['packages'] = []
+            for pkg in self.descriptor['install']:
+                self.descriptor['packages'].append(ListItem(pkg))
+        elif 'packages' in self.descriptor:
```

Comment:
This is the "old" way, right? Maybe print a warning?

Reply:
the idea is, that both approaches are valid. So you can still write simple way, if you dont need to override.

Reply:
Then we need to have it documented. Very well documented.